wordpress-mobile/WordPressKit-iOS
# Comments: add REST test for successful fetching

This adds a test for successful REST Comment fetching. I'll add more tests and XMLRPC tests in follow ups.

### Testing Details

- Run `CommentServiceRemoteRESTTests`. Verify it passes.
- Change any of the assert values. Verify the test fails.
